testsuite/src/it/java/org/graalvm/tests/integration/utils/GDBSession.java

```diff
@@ -272,7 +272,7 @@ public CP[] get(boolean inContainer) {
         }
     };
 
-    private static final CP SHOW_VERSION = new CP("show version\n", Pattern.compile(".*", Pattern.DOTALL));
+    private static final CP SHOW_VERSION = new CP("show version\n", Pattern.compile(".*GNU.*", Pattern.DOTALL));
```

Comment:
Shall we use `gdb` over `GNU`? We expect a `gdb` version after all.

Reply:
It should be reasonably safe to use `gdb`, the reason I chose GNU is because GDB is licensed under GPL and thus it's required to show a notice about the license in interactive mode:

> If the program does terminal interaction, make it output a short notice like this when it starts in an interactive mode:

Checking https://www.gnu.org/licenses/gpl-3.0.html though I see that it's not mandatory to mention the license itself so `GNU` might not be the best choice after all... `NO WARRANTY` on the other hand... :)

In any case, let's go with gdb and hope that maintainers keep it in the version string in all distributions.